src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -94,7 +96,7 @@ func logPanic(msg string) {
 		panic("Error initializing logging")
 	}
 
-	<-logging.Configure("", "", cfg.InstanceId, version, revisionDate)
+	<-logging.Configure("", "", "", version, revisionDate)
```

Comment:
We still need the instanceId for logging. `logging` package can also retrieve instance id by itself.